extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeFeedExtractor.java

```diff
@@ -57,19 +61,44 @@ public ListExtractor.InfoItemsPage<StreamInfoItem> getInitialPage() {
     @Nonnull
```

Comment:
Mmmh, this method is marked as Nonnull, but are nulls turned into empty strings?

Reply:
If we couldn't extract the URL, an empty string is provided, nulls are just handled to avoid `NullPointerException`s if there is no element found in the path we provide to jsoup (the [`first` method of the `Elements` class returns `null`](https://jsoup.org/apidocs/org/jsoup/select/Elements.html#first()) in this case).